http_proxy.go

```diff
@@ -10,11 +10,11 @@ import (
 
 	"github.com/getlantern/http-proxy/commonfilter"
 	"github.com/getlantern/http-proxy/filters"
-	"github.com/getlantern/http-proxy/forward"
 	"github.com/getlantern/http-proxy/httpconnect"
 	"github.com/getlantern/http-proxy/listeners"
 	"github.com/getlantern/http-proxy/logging"
 	"github.com/getlantern/http-proxy/server"
+	"github.com/getlantern/http-proxy/stateful"
```

Comment:
Yeah, I don't love "forward" either. How about "pforward" for "persistent" forward as in "persistent HTTP connection"?